cachito/workers/config.py

```diff
@@ -45,6 +45,7 @@ class Config(object):
     cachito_nexus_max_search_attempts = 5
     cachito_nexus_npm_proxy_url = "http://localhost:8081/repository/cachito-js/"
```

Comment:
I'm little confused about the configuration here. Why is this in the base `Config` class while the PyPI proxy URL is in the `DevelopmentConfig`? Also, I'm curious about what's the difference between `http://nexus` URL and `http://localhost`, since it looks like both of these work.

Reply:
(for pip and now ruby, we do directly download stuff from the proxy url)

Reply:
Oh, okay, I'm going to keep RubyGems config in `DevelopmentConfig` then (as pip does).